examples/ui-demo/src/state/index.tsx

```diff
@@ -0,0 +1,170 @@
+"use client";
+
+import { alchemyConfig, Config, queryClient } from "@/app/config";
+import { AuthCardHeader } from "@/components/shared/AuthCardHeader";
+import { AlchemyClientState } from "@account-kit/core";
+import {
+  AlchemyAccountProvider,
+  AlchemyAccountsUIConfig,
+  AuthType,
+  useUiConfig,
+} from "@account-kit/react";
+import {
+  createContext,
+  Dispatch,
+  PropsWithChildren,
+  SetStateAction,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { useStore } from "zustand";
+import {
+  createDemoStore,
+  generateClassesForRoot,
+  generateStylesForRoot,
+} from "./store";
+
+export type ConfigContextType = {
+  config: Config;
+  setConfig: (partial: Partial<Config>) => void;
+  nftTransfered: boolean;
+  setNFTTransfered: Dispatch<SetStateAction<boolean>>;
```

Comment:
Typo: "transfered" -> "transferred", in these two lines, likewise in the `useState` later.